library/testing/TestSDKMultiColoring.cxx

```diff
@@ -10,7 +10,7 @@
 
 int TestSDKMultiColoring(int argc, char* argv[])
 {
-  f3d::log::setVerboseLevel(f3d::log::VerboseLevel::DEBUG);
+  f3d::log::setVerboseLevel(f3d::log::VerboseLevel::ERROR);
```

Comment:
This is needed otherwise the baseline is different with EGL and OSMesa because a warning is triggered with X failing

Reply:
Why would still test trigger a X failure though ?

Reply:
This test is always running with an `auto` rendering backend so the X failure is triggered when the test is running on the EGL/OSMesa CI jobs

Reply:
this test has nothing different than other SDK tests, I still dont get why it would trigger this behavior and not other tests

Reply:
there's not only this one, 4 tests have the same behavior.